src/index.ts

```diff
@@ -0,0 +1,126 @@
+import { Reshuffle, BaseConnector, EventConfiguration } from 'reshuffle-base-connector'
+import ImapClient from 'imap'
+import { simpleParser } from 'mailparser'
+
+export interface IMAPConnectorConfigOptions {
+  host: string
```

Comment:
Are they all mandatory?
What is the markSeen option? shouldn't be an action?

Reply:
`markSeen` is part of IMAP.
Changed a few to be non-mandatory